sql/src/main/java/org/apache/druid/sql/calcite/run/EngineFeature.java

```diff
@@ -36,6 +42,11 @@
    */
   CAN_RUN_TOPN,
 
+  /**
+   * Queries of type {@link org.apache.druid.query.timeboundary.TimeBoundaryQuery} are usable.
+   */
+  CAN_RUN_TIME_BOUNDARY,
```

Comment:
Nit: Since these items are logically part of a feature set, maybe just use the feature name itself. TOPN_QUERY, SELECT, TIME_BOUNDARY_QUERY, etc.